graql/language/match.feature

```diff
@@ -1505,7 +1509,7 @@ Feature: Graql Match Query
     Given graql define
       """
       define
-     unit sub attribute, value string, owns unit, owns ref;
+      unit sub attribute, value string, owns unit, owns ref;
```

Comment:
Client Python's `behave` framework also crashed attempting to parse this string before this change.

Reply:
lol